api/client-testextension/src/main/java/org/projectnessie/client/ext/NessieClientResolver.java

```diff
@@ -158,17 +159,19 @@ public NessieApiVersion apiVersion() {
     @jakarta.annotation.Nonnull
     @Override
     public NessieApiV1 make(NessieClientCustomizer customizer) {
-      HttpClientBuilder clientBuilder = HttpClientBuilder.builder().withUri(resolvedUri);
+      NessieClientBuilder clientBuilder =
+          createClientBuilderFromSystemSettings().withUri(resolvedUri);
```

Comment:
Do you think we could add a new interface similar to `NessieClientUriResolver` to allow tests to choose the client name in runtime? Defaulting to system properties may be too restrictive, I guess.

Reply:
done

Reply:
thx :)